ocaml/rrdd/rrdd_server.ml

```diff
@@ -628,12 +628,18 @@ module Plugin = struct
 			verify_checksum_freshness ~uid ~checksum;
 			parse_payload ~json:payload
 		with e ->
-			error "Failed to process plugin: %s (%s)" uid (Printexc.to_string e);
-			log_backtrace ();
+			let log e =
+				error "Failed to process plugin: %s (%s)" uid (Printexc.to_string e);
+				log_backtrace ()
+			in
 			match e with
-			| Invalid_header_string | Invalid_length | Invalid_checksum
-			| No_update as e -> raise e
-			| _ -> raise Read_error
+			| No_update -> raise No_update
```

Comment:
Where is this exception eventually handled?    Will it bubble up to the handler in Rrdd_main.monitor_loop?   I don't see any evidence of that happening in the logs, but I can't see anywhere else where exceptions from here should be handled.